### ISSUE TITLE
Wings stop working after some playtime with them

### ISSUE THREAD

**Issue by Veneiro** [adriantodt/Winged#39]
I was playing like usual and suddenly after dying by an archer the wings stopped working. I have them on my back but I cannot glide. I also even try to remove my wings and craft them again but I cannot use them neither

**Comment by NotJustAnna** [comment]
This is a FallFlyingLib bug, which (hopefully) be superseded by an official Fabric API module: https://github.com/FabricMC/fabric/pull/1815

Once the module is merged, I'll work on a version that uses the Fabric API module and this will (hopefully²) be fixed.
